### DISCUSSION TITLE
wasmer.io is worse and broken, bring back wapm.io

### DISCUSSION THREAD

**Discussion by comods** [wasmerio/wasmer#5330] [General]
wapm.io was great, worked, simple, and ran embedded runtimes.
wasmer.io is way worse, broken, confusing, ugly, and unusable.

Where is the mobile type-able url that takes you to a clean package search? Used to be wapm.io (7 chars) and now there isn't one.

- [ ] wasmer.io/explore (17 chars) is 10 characters longer then it used to be.

It looks and runs like it was made by an outdated AI prompt:
- [ ] Takes up a whole monitor to show ONLY 4 links.
- [ ] You can't click on a link's card, you have to click exactly on the small text of the huge card (The reason mobile has huge buttons is for larger touch hitboxes not looks).
- [ ] None of the tabs work (Quick sites, Widgets, Databases, CLI toys, Programming languages).
- [ ] The Titles (Recently published, Staff picks) do not have the same vertical styling.
- [ ] Embedded webassembly.sh no longer works.
- [ ] Embedded WASM-4 / github.com/wasmerio/io-devices-lib no longer works.
- [ ] Sometimes says "(8 results)" with "Your search did not match anything in our database"
- [ ] When you select the search bar, the popup menu is completely broken. For a moment shows wrong styling. If you move your mouse down to select an item it will resize and if it shrunk enough to move your mouse off the edge it will close itself. If you try to click the lowest item it always resizes and never clicks the item.

Your CLI tools are great, just please give us a working website, like wapm.io was.

Focus on getting traffic to packages or no one has a use for wasmer.io/explore.

**Comment by syrusakbary** [comment]
Thanks a lot for the great feedback @comods . It's true that packages have been not getting a lot of love since uniting the both websites. We'll work on those things in the coming quarter, please stay tuned and keep the feedback coming. It helps us immensely
